Riot/Modules/Room/RoomViewController.m

```diff
@@ -1611,7 +1611,8 @@ - (BOOL)supportCallOption
     {
         return NO;
     }
-    BOOL callOptionAllowed = (self.roomDataSource.room.isDirect && RiotSettings.shared.roomScreenAllowVoIPForDirectRoom) || (!self.roomDataSource.room.isDirect && RiotSettings.shared.roomScreenAllowVoIPForNonDirectRoom);
+    BOOL callOptionAllowed = (self.roomDataSource.room.isDirect && RiotSettings.shared.roomScreenAllowVoIPForDirectRoom)
+    || (!self.roomDataSource.room.isDirect && RiotSettings.shared.roomScreenAllowVoIPForNonDirectRoom && self.mainSession.vc_homeserverConfiguration.jitsi.serverURL);
```

Comment:
roomScreenAllowVoIPForNonDirectRoom set to false is the same as setting Jitsi URL to nil

maybe we should not even test Jitsi here and let it go to the error message ? (and then see the change on Jitsi URL only as a security to make sure that no broken UI could start an unwanted Jitsi call)
=> this could be useful as well if one wants to implement a replacer for Jitsi call